src/main/java/task/TaskListTest.java

```diff
@@ -0,0 +1,26 @@
+package task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    void addTask() {
+        TaskList tl = new TaskList();
+        assertEquals("Ah sure. I've added this task:\n" +
+                "      [T][ ] sleep\n" +
+                "    Now you have 1 task in the list.", tl.addTask(new Todo("sleep"), 1));
```

Comment:
Spacing on this line should be 12 instead of 8 as it is a wrapped line. Same for below case under method removeTask()